src/providers/extraction_providers/visionExtractionProvider.ts

```diff
@@ -45,12 +43,12 @@ export class VisionExtractionProvider implements ExtractionProvider {
 		const client = new vision.ImageAnnotatorClient();
 		const request = {
 			image: { source: { imageUri: image.source } },
-			features: [{ type: requestTypes.labelDetection }, { type: requestTypes.objectLocalization }],
+			features: [{ type: requestTypes.LabelDetection }, { type: requestTypes.ObjectLocalization }],
 		};
 
 		const [results] = await client.annotateImage(request);
-		const labels = results.labelAnnotations!;
-		const objects = results.localizedObjectAnnotations!;
+		const labels = results.labelAnnotations || [];
+		const objects = results.localizedObjectAnnotations || [];
```

Comment:
Is there a possibility where `results` is null or undefined

Reply:
results is only used by the two lines directly following it, so it should be safe in the case that it is null or undefined